src/lib/utils/numericUuid.ts

```diff
@@ -0,0 +1,4 @@
+function numericUuid() {
+    return Math.round(Math.random() * 10000000000000000);
```

Comment:
A number is okay, but then of course we must not call it uuid because it is no uuid (https://en.wikipedia.org/wiki/Universally_unique_identifier)